src/main/java/controllers/TrainerController.java

```diff
@@ -54,4 +57,36 @@ public TrainerData getTrainer(String userIdentifier) throws UserDoesNotExistErro
 
         return trainer;
     }
+
+    /**
+     * Searches for all users with the given first name.
+     * @param firstName The first name of the member.
+     * @return A list of all the members with the given first name.
```

Comment:
I like how this returns a list of members! Good design choice. I didn't consider returning multiple members with the same name.